lib/SILGen/SILGenExpr.cpp

```diff
@@ -1590,6 +1590,10 @@ ManagedValue emitCFunctionPointer(SILGenFunction &SGF,
   // C function pointers cannot capture anything from their context.
   auto captures = SGF.SGM.Types.getLoweredLocalCaptures(constant);
 
+  // Catch cases like:
+  //   func g(_ : @convention(c) () -> ()) {}
```

Comment:
I just want to say. Thank you so much for putting stuff like this in here! Very useful! = ).